ros/api/v1/hosts.py

```diff
@@ -45,7 +43,7 @@ def get(self):
             .distinct()
```

Comment:
Just a small nitpick - we can remove `distinct` from here as well

Reply:
Done!